### DISCUSSION TITLE
Account got flagged

### DISCUSSION THREAD

**Discussion by ojous007** [community/community#48633] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

When i came to know that i could have a pro account on GitHub with my student ID, I made a student account of GitHub,  but as soon as I made my account, my account got flagged and even after asking the support staff and others I was not able to get any help, so I want you all to help me out with this flag shit.

**Comment by cehfisher** [comment]
Hi @ojous007 - It looks like your issue was mistakenly added to the accessibility discussion board. @queenofcorgis could you please move this to the correct category? Thanks!

**Comment by queenofcorgis** [comment]
Hi @ojous007, 

This community is managed by GitHub employees, but it is not a support forum and there is no SLA or guarantee each post will get a response from a community manager or product leader.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
